src/AspNetCore/OData/src/Asp.Versioning.OData/Asp.Versioning.OData.csproj

```diff
@@ -15,7 +15,7 @@
  </ItemGroup>
 
  <ItemGroup>
-  <PackageReference Include="Microsoft.AspNetCore.OData" Version="[8.0.2,9.0.0)" />
+  <PackageReference Include="Microsoft.AspNetCore.OData" Version="[8.0.2,9.3.0)" />
```

Comment:
Major versions are the boundary. This should be `[8.0.2,10.0.0)` which reads as:

```
8.0.2 <= version < 10.0.0
```

For historical context, there were breaking changes released in early patches of OData 8.0. `8.0.2` what the minimum required version with fixes that were released with options for people to navigate other changes afterward.